### ISSUE TITLE
Question: key shortcuts?

### ISSUE THREAD

**Issue by rafi-d** [1Remote/1Remote#658]
Hi,
Is there a list of key shortcuts available for 1remote manager? 
Specifically, I like the shortcut for "**reconnect**" menu item (for the selected PC[s]), for my frequent use case:
The remote is locking the session into the login screen, and re-connect is the easiest way to re-login... 

...and If possible - for connect too?

There was an old request here for user-definable hot keys, but the above is the minimum for me :) ...

**Comment by majkinetor** [comment]
There is also #405.

**Comment by majkinetor** [comment]
> BTW, emulating key/mouse input/juggle option might work for preventing timeouts if you can generate such inputs

Good idea, it never occurred to me. Emulating keys is IMO too intrusive, as that can influence the remote. Moving a mouse, though, slightly, then moving it in the opposite direction could work.

**Comment by rafi-d** [comment]
> Emulating keys is IMO too intrusive, as that can influence the remote. Moving a mouse, though, slightly,

Right, possibly for a "timeout" duration... It might work, but needs testing, since it is not clear if a mouse by itself, without any action (like scroll)  will count as an "activity" ... 
A key pressed is always an activity, so maybe some "harmless" key-code can be found ..

**Comment by majkinetor** [comment]
BTW, I think reconnect hotkey might not be possible - once remote host grabs input, its AFAIK not possible to do so (see #40). Since after locking out you remote host will still keep keyboard focus, those hotkeys would finish on remote.
